ftf_cli/operations.py

```
@@ -0,0 +1,206 @@
+
+    # Create zip file
+    zip_path = None
+    try:
+        zip_path = create_module_zip(path)
+
+        # Prepare authentication
+        auth_string = base64.b64encode(f"{username}:{token}".encode()).decode().strip()
+        headers = {"Authorization": f"Basic {auth_string}"}
+
+        # Prepare files for upload
+        files = {"file": ("module.zip", open(zip_path, "rb"), "application/zip")}
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)